crates/aptos/README.md

```diff
@@ -3,17 +3,25 @@
 The `aptos` tool is a command line interface (CLI) for debugging, development, and node operation.
 This document describes how to install the `aptos` CLI tool and how to use it.
 ## Installation
-### Install Cargo
+### Install precompiled binary (easy mode)
+* Navigate to the [release page](https://github.com/aptos-labs/aptos-core/releases) for Aptos CLI.
+* Download the latest release for your computer.
+* Place this at a location for you to run it e.g. `~/bin/aptos` in Linux.
+* On Linux and Mac, make this executable `chmod +x ~/bin/aptos`.
+* Now type `~/bin/aptos help` to read help instructions.
+* If you want you can add `~/bin` to your path in your appropriate `.bashrc` or `.zshrc` for future use
+
+### Install Cargo (harder mode)
```

Comment:
For our "mini-hackathon" (before the binary was published) I found it useful to build the binary from a local git checkout.